.github/workflows/ci.yml

```
@@ -463,7 +463,10 @@ jobs:
       - name: Set up gdal-bin and sqlite3-tools
         if: matrix.os == 'ubuntu-latest'
         run: sudo apt-get update && sudo apt-get install -y gdal-bin sqlite3-tools
-
+      - name: Install sqlite3 and sqldiff on Windows
+        if: matrix.os == 'windows-latest'
+        run: choco install sqlite --yes
+        shell: powershell      
```

To make pre-commit happy

```suggestion
      - name: Install sqlite3 and sqldiff on Windows
        if: matrix.os == 'windows-latest'
        run: choco install sqlite --yes
        shell: powershell
```